packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -306,6 +312,24 @@ export const esmPatientRegistrationSchema = {
       _default: '736e8771-e501-4615-bfa7-570c03f4bef5',
     },
   },
+  MPI: {
+    title: {
+      _type: Type.String,
+      _default: 'MPI',
+    },
+    baseAPIPath: {
+      _type: Type.String,
+      _default: '/ws/fhir2/R4/MPIPatient',
+    },
+    preferredPatientIdentifierTitle: {
+      _type: Type.String,
+      _default: 'Health ID',
+    },
+    preferredPatientIdentifierType: {
```

Comment:
I think that this type of configuration probably belongs as part of the backend settings, not as part of the frontend configuration.

Reply:
Nice one. I believe there could be an existing GP; guess I was too dumb to think about it.

Reply:
Question: How can we best represent these in the backend? 

Notes:
We basically want to be able to map identifiers basing on 2 or 3 orientations: 'internal-preferred' | 'internal' | 'external'. 

1. 'internal-preferred': represents the primary identifier: Currently configurable through a metadata mapping: `metadatamapping/termmapping?code=emr.primaryIdentifierType` (I'm guessing it's controlled by some GP) 
2. 'internal': represents extra or secondary OpenMRS identifiers
3. 'external': represents the identifier assigned to the patient by the MPI

@ibacher Would it be ideal to configure the 'external' identifier-type through a Global Property declared by the CR module?

Reply:
> I'm guessing it's controlled by some GP

It's actually controlled by a metadata mapping.

I don't love using a GP for this, because it's fundamentally not global, i.e., the external identifier (presumably) varies with the MPI, and using a GP for this kind of restricts us to a MPI-per-system. I'd tend to prefer just having a table to store that configuration data...

Reply:
This means we would need an extra endpoint to fetch this config? Should we setup a technical call with the folks working on the CR module? Btw, it would be great if you added some technical notes to the epic => https://issues.openmrs.org/browse/HIE-1

cc: @smallgod, @pmanko

Reply:
Might be good to have a call about this. I'm not 100% sure that we actually need the frontend to know about much of this, i.e., it's not directly querying the MPI, but rather a backend operation that queries the MPI, so we can mark the MPI-provided identifier in that response, if it's necessary to differentiate them. Basically, I'm not quite sure I understand why we need this distinction in the frontend, I'm sure I'm missing something.

Reply:
The front-end needs to be able to classify records into:
1. Internal (Not synchronised with the MPI)
2. External (Record from the MPI and not synchronised with the EMR)
3. Cross cutting (Records synchronised with both systems)

It achieves the above requirements through patient identifications

Reply:
But it should be the backend directly communicating with the MPI, right?

Reply:
Yes, its the backend in communication with the MPI